modules/electrophysiology_browser/php/models/electrophysiofile.class.inc

```diff
@@ -86,10 +86,10 @@ class ElectrophysioFile implements \LORIS\Data\DataInstance
     /**
      * Implements \LORIS\Data\DataInstance interface
      *
-     * @return string
+     * @return array which can be serialized by json_encode()
      */
-    public function toJSON(): string
+    public function jsonSerialize() : array
     {
-        return '';
+        return [];
```

Comment:
This one doesn't look right. Why does it turn from the empty string which returns nothing to the caller to the empty array (which is an actual value)? I think the type signature is just wrong for this file and it shouldn't be returning an array.